Schweizerisches Bundesgericht.js

```diff
@@ -0,0 +1,243 @@
+ *
+ * @param {Document} doc
+ * @param {{}}       parameters
+ * @returns {{}}
+ */
+function getSearchResultsNews(doc, parameters) {
+	let domRows = doc.querySelectorAll("div.eit>table:nth-of-type(2)>tbody>tr");
+	let items = {};
+
+	for (let i = 1; i < domRows.length; i += 2) {
+		let date = domRows[i].children[1].innerText;
+		let url  = domRows[i].children[2].children[0];
+		let number = domRows[i].children[2].children[0].innerText;
+		let subject = domRows[i].children[4].innerText;
+		items[url] = number + " (" + date + "; " + subject + ")";
+	}
+
+	return items;
+}
+
+/**
+ * collect items from a search result page of all rulings
+ *
+ * Those pages are found at:
+ * /ext/eurospider/live/de/php/aza/http/index.php?l...type=simple_query|simple_similar_documents
+ *
+ * @param {Document} doc
+ * @param {{}}       parameters
+ * @returns {{}}
+ */
+function getSearchResultsBGer(doc, parameters) {
+	let domRows = doc.querySelectorAll("div.ranklist_content>ol>li");
+	let items = {};
+
+	for (let i = 1; i < domRows.length; i++ ) {
+		let meta = domRows[i].children[0].children[0].innerText;
+		let date = meta.substring(0, meta.indexOf(' '));
+		let number = meta.substring(date.length + 1);
+		number = number.substring(0, 2) + "_" + number.substring(3);
+		let url  = domRows[i].children[0].children[0];
+		let subject = domRows[i].children[2].children[1].innerText;
+		let subject2 = domRows[i].children[2].children[2].innerText;
+		items[url] = number + " (" + date + "; " + subject + ": " + subject2 + ")";
+	}
+	return items;
+}
+
+/**
+ * collect items from a search result page of formally published rulings
+ *
+ * Those pages are found at:
+ * /ext/eurospider/live/de/php/clir/http/index.php?...type=highlight_simple_query|highlight_simple_similar_documents
+ *
+ * @param {Document} doc
+ * @param {{}}       parameters
+ * @returns {{}}
+ */
+function getSearchResultsBGE(doc, parameters) {
+	let dom_rows = doc.querySelectorAll("div.ranklist_content>ol>li");
+	let items = {};
+
+	for (let i = 1; i < dom_rows.length; i++ ) {
+		let url = dom_rows[i].children[0].children[0];
+		let number = abbrevPublished[parameters._lang] + url.innerText;
+		let subject = dom_rows[i].children[2].children[1].innerText;
+		subject = subject.substring(subject.indexOf(']') + 2);
+		items[url] = number + " (" + subject + ")";
+	}
+	return items;
+}
+
+function _scrape(doc, url, parameters) {
+		let item = new Zotero.Item("case");
+
+		item.url = url;
+		item.court = courtName[parameters._lang];
+		item.attachments.push({
+			title: "Snapshot",
+			document: doc
+		});
+		if (parameters._collection === "BGE") {
+			let docket = parameters.highlight_docid;
+			docket = docket.substring(6, docket.length - 3).replace(/-/g, ' ');
+			item.title = abbrevPublished[parameters._lang] + docket;
+			item.number = item.title;
+			item.date = parseInt(docket.split(' ')[0]) + 1874;
+			item.abstractNote = ZU.trimInternal(doc.querySelector("div#regeste>div.paraatf").innerText);
```

Comment:
```suggestion
			item.abstractNote = ZU.trimInternal(text(doc, "#regeste .paraatf"));
```

Reply:
Any reason why this suggestion wouldn't work?

Reply:
Sorry, I seem to have missed some changes while commiting... Corrected.